src/scss/includes/menu.scss

```diff
@@ -2,7 +2,9 @@
 #react_menu__container {
   align-self: stretch;
```

Comment:
If you turn this element to `position: absolute`, this rule related to `flex` shouldn't have any effect anymore.